src/Sarif/CodeGenHints.json

```diff
@@ -114,8 +114,8 @@
       "arguments": {
         "typeName": "AlgorithmKind",
         "description": "Values specifying different hashing algorithms.",
-        "zeroValue": "Unknown",
-        "enumValues": [
+        "zeroValueName": "Unknown",
+        "enumNames": [
```

Comment:
I changed these names when I introduced the `enumValues` argument (see below) to specify the numeric values.